website/snippets/_install-dbt-extension.md

```diff
@@ -112,64 +112,121 @@ This will start the upgrade tool and guide you through the Fusion upgrade with a
+### First-time registration
 
-### New user registration
+Use this if you've _never_ created a dbt account before. What you'll do: Open the command, enter your name and email, verify the email, and you're done 🎉!
 
-If you do not already have a dbt account, you'll need to get registered. This only takes a minute!
-1. Click the registration prompt in your editor.
-     <Lightbox src="/img/docs/extension/registration-prompt.png" width="70%" title="The extension registration prompt in VS Code."/>
-2. Accept any prompts to open the link in your browser.
-3. Fill out the registration form, then click **Continue**.
-    <Lightbox src="/img/docs/extension/registration-screen.png" width="70%" title="The extension registration page in the browser."/>
-4. You will receive an email with a verification link. Once you click it, your registration is complete!
+1. Click the registration prompt or open the command palette (Ctrl + Shift + P (Windows/Linux) or Cmd + Shift + P (macOS)) and run: **dbt: Register dbt extension**.
+   <Lightbox src="/img/docs/extension/registration-prompt.png" width="70%" title="The extension registration prompt in VS Code."/>
+2. In the browser registration form, enter your name and email, then click **Continue**.
+3. Check your inbox for a verification email and click the verification link.
+4. After verification, return to the browser flow to complete sign‑in.
+5. You'll return to the editor and see **Registered**.
+6. Continue with the [Get started](#getting-started) onboarding workflow and get your dbt project up and running.
 
-### Accessing your dbt account
+**Note:** You do not need a <Constant name="dbt_platform" /> project to register; this only creates your dbt account.
 
-Registering to use the dbt extension makes it easy to create a full dbt account. You can follow these
-steps to finish setting up your account (_Note: This is not required to use the dbt extension_).
+---
 
-1. Navigate to [us1.dbt.com](https://us1.dbt.com) and click **Forgot password?**.
-2. Enter the email address you used for your dbt extension registration and click **Continue**.
-3. Check your email for a verification link and follow the password reset instructions to set a password for your account.
+### Existing account sign-in {#existing-dbt-account}
 
-Now that you have activated your dbt developer account, you can access features of the <Constant name="dbt_platform" />. You can also re-download your registration key using the steps outlined in [Register with an existing dbt account](#register-with-an-existing-dbt-account) if you need to set up the dbt extension on a new machine.
+Use this if you have an existing dbt account &mdash; including older or inactive accounts.
 
-### Register with an existing dbt account 
-<!-- This anchor is linked from the VS Code registration page. Please do not change it -->
+What you'll do: Start registration → sign in through the OAuth flow → return to VS Code.
```

Comment:
I think we can remove this. We don't usually give a recap of steps before them and it seems excessive.
```suggestion
```